dom-events/buttons.js

```diff
@@ -0,0 +1,17 @@
+function handleClick(event) {
+  console.log('button clicked');
+}
+var clickButton = document.querySelector('.click-button');
+clickButton.addEventListener('click', handleClick);
+
+function handleMouseover(event) {
+  console.log('button hovered');
+}
+var hoverButton = document.querySelector('.hover-button');
+hoverButton.addEventListener('mouseover', handleMouseover);
+
+function handleDoubleClick(event) {
+  console.log('button double-clicked');
+}
+var doubleClicked = document.querySelector('.double-click-button');
+doubleClicked.addEventListener('dblclick', handleDoubleClick);
```

Comment:
You are missing your logs for the `event` object and `event.target` in each event handler function 👀